README.md

```diff
@@ -328,6 +328,256 @@ npm install prosemirror-utils
    ```
 
 
+ * **`moveRow`**`(originRowIndex: number, targetRowIndex: targetColumnIndex, options: ?MovementOptions) → fn(tr: Transaction) → Transaction`\
+   Returns a new transaction that moves the origin row to the target index;
+
+   by default "tryToFit" is false, that's means if you try to move a row to a place
+   where we will need to split a row with cell merges it'll blow an exception, for example:
```

Comment:
* `with cell merges` -> `with merged cells`
* `it'll blow an exception` -> `it'll throw an exception`